src/data/actions/courseInfo.js

```diff
@@ -118,6 +118,12 @@ function updateFormValuesAfterSave(change, currentFormValues, initialValues) {
         per_click_usa: perClickUSA,
         per_click_international: perClicknternational,
       },
+      additional_metadata: {
+        certificate_info: {
+          heading,
+          blurb,
+        },
+      },
```

Comment:
make sure js-doc for this method is updated to include this.

Reply:
addressed!